pkg/collector/corechecks/servicediscovery/impl_linux.go

```diff
@@ -21,7 +21,7 @@ func init() {
 }
 
 type linuxImpl struct {
-	getDiscoveryServices func(client *sysprobeclient.CheckClient) (*model.ServicesResponse, error)
+	getDiscoveryServices func(client *sysprobeclient.CheckClient) (*model.CheckResponse, error)
```

Comment:
It would have been nice to have kept this rename outside of this PR since it's not strictly necessary.